<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExaminationListWindow</class>
 <widget class="QMainWindow" name="ExaminationListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>檢驗查詢</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget_examination_list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ExaminationKey</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>檢驗日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>病歷號</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>病患姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>醫事檢驗所</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>醫檢師</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_examination"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="action_open_record"/>
    <addaction name="action_delete_record"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編輯</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_requery"/>
   <addaction name="separator"/>
   <addaction name="action_open_record"/>
   <addaction name="action_delete_record"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <action name="action_open_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-open.svg</normaloff>../icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>調閱報告</string>
   </property>
   <property name="toolTip">
    <string>調閱報告</string>
   </property>
  </action>
  <action name="action_delete_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/user-trash.png</normaloff>../icons/user-trash.png</iconset>
   </property>
   <property name="text">
    <string>刪除報告</string>
   </property>
  </action>
  <action name="action_requery">
   <property name="icon">
    <iconset>
     <normaloff>../icons/reload.svg</normaloff>../icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>重新查詢</string>
   </property>
   <property name="toolTip">
    <string>重新查詢病歷</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>pushButtonQueryClicked()</slot>
  <slot>pushButtonRequeryClicked()</slot>
  <slot>tableWidgetCellDoubleClicked()</slot>
  <slot>actionOpenCaseTriggered()</slot>
  <slot>actionDeleteCaseTriggered()</slot>
  <slot>actionPrintPresTriggered()</slot>
 </slots>
</ui>
